src/index.html

```diff
@@ -1,13 +1,61 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body>
+    <section class="card" data-qa="card">
+      <img
+        class="card__mac-image"
+        src="images/imac.jpeg"
+        alt="Mac image"
+      >
+
+      <div class="card__description">
+        <p class="card__description-name">
+          Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)
+        </p>
+
+        <p class="card__description-id">
+          Код товара: 195434
+        </p>
+
+        <div class="card__description-rating">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+
+          <p class="comments">
```

Comment:
I would think that this element is a part of a block and you should give a right name to it's class
```suggestion
          <p class="card__discription__comments">
```

Reply:
`card__discription__comments` - це елемент елемента, такі конструкції заборонені методологією BEM